docs/advanced/helpers-components.md

```diff
@@ -0,0 +1,34 @@
+---
+id: helpers-components-helpers
+title: Components
+---
+
+Components helper is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Components` namespace.
```

Comment:
> To extend it, use `Eightshift_Libs\Helpers\Components` namespace

Not sure what this means. Could you just write it's namespace if that's the information you were going for?

Reply:
This should be `class` in all the instances where it appears. I think I commented this, but it was merged before my comments...

Reply:
Like @dingo-d said it should be `Eightshift_Libs\Helpers\Components` class. 
@dingo-d your fixed were done on the other pages. This is a new page. I will fix it